backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/migration/MigrationTaskDTO.java

```diff
@@ -0,0 +1,40 @@
+package org.apache.james.backends.cassandra.migration;
+
+import java.util.function.Function;
+
+import org.apache.james.backends.cassandra.versions.SchemaVersion;
+import org.apache.james.json.DTOModule;
+import org.apache.james.server.task.json.dto.TaskDTO;
+import org.apache.james.server.task.json.dto.TaskDTOModule;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.annotations.VisibleForTesting;
+
+class MigrationTaskDTO implements TaskDTO {
+
+    @VisibleForTesting static final Function<MigrationTask.Factory, TaskDTOModule<MigrationTask, MigrationTaskDTO>> MODULE =
```

Comment:
Why only visible for testing ? We will use it for the serialization in the event store.

Reply:
Maybe, yes, but I don't need more right now.